VTEX - Reviews and Ratings API.json

```diff
@@ -13,14 +13,14 @@
             "variables": {
```

Comment:
Linha 11:
A url deve ser apenas `https://{accountName}.{environment}.com`. A parte `/reviews-and-ratings/api` deve ser incluída em todos os paths.